### PR TITLE
implementation of a simple REST GET/POST server using Django

### PR DESCRIPTION
**Pull request by RajsimmanRavi**

### CODE REVIEW

File: interview/server/views.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@csrf_exempt
+@require_http_methods("POST")
+def insert_messages(request):
+    try: 
+        data = json.loads(request.body)
+        message = str(data["message"])
+        sender = str(data["sender"])
+        conversation_id = str(data["id"])
+        created = datetime.now()
+        if len(Conversation.objects.filter(conversation_id=conversation_id)) > 0:
```
Comment (noqcks) [inline interview/server/views.py]: It might have not been as explicitly clean in the instructions, but a conversation should be able to hold more than 1 message. https://github.com/AdaSupport/backend-challenge#conversationsconversation_id-resource

File: interview/server/views.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    def datetime_to_str_converter(o):
+        if isinstance(o, datetime):
+            return o.__str__()
+    try:
+        return_object = Conversation.objects.get(conversation_id=id)
+        return_dict = model_to_dict(return_object) # for returning properly
+        return_json_dict = json.dumps(return_dict, default = datetime_to_str_converter) # need to convert the timestamp to string
+        data = {'type': 'ok', 'data': json.loads(return_json_dict)}
+        return JsonResponse(data)
+    except Exception as e:
```
Comment (noqcks) [inline interview/server/views.py]: I think I'm running into an unexpected error here: 
```
~ curl -H "Accept: application/json" http://127.0.0.1:8000/conversations/1235
{"type": "error", "message": "get() returned more than one Conversation -- it returned 2!"}⏎
```

File: interview/server/admin.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (noqcks) [inline interview/server/admin.py]: Is this file still needed?

**Comment by noqcks** [comment]
Hi @RajsimmanRavi 

First of all thank you for taking the time to complete this. I was impressed by the quick turnaround time, but I think there were a few small errors that could have been avoided. 

Nits:
- __pycache__ should be .gitignored
- dependencies should be added to a requirements.txt or a Pipfile

I'll follow up via email.

**Comment by RajsimmanRavi** [comment]
Sorry Benji, 

I should have been more careful regarding those issues. I will remember them next time for sure. Thanks for letting me know. 

Thanks,

Raj.
